### ISSUE TITLE
feature request: explicit div in output

### ISSUE THREAD

**Issue by bitencode** [prettier/plugin-pug#241]
## Request / Idea

Option to not remove the `div` element when literal class or id is used.
I recently found your plugin and it's just about perfect for our needs (Vue with Pug).  However, for reasons we don't allow the use of `.my-class` or `#myId` in our templates.  We require the use of `div.my-class` particularly because it is hard for new devs to grok and it's more explicit.

## Input

```pug
div.my-class(attrs)
  div(class="anohterClass")
```

## Expected Output

```pug
div.my-class(attrs)
  div.anotherClass
```
instead this is always produced:
```pug
.my-class(attrs)
  .anotherClass
```

While that is perfectly valid pug syntax we don't allow it because we believe it's not a good feature.

## Additional Context
@Shinigami92 Would you be open to  a PR that added a pug option to allow the `div` element to remain in the output?
If so, do you have an option name preference? I was thinking of `pugExplicitDiv`, but will use your preference.

I started looking at this and it seems like it would be fairly easy to create.

**Comment by Shinigami92** [comment]
This will land in the next release, but I have some other PRs that also needs to be merged
Maybe I will release a beta in the next few days

**Comment by bitencode** [comment]
Thank you @Shinigami92

**Comment by Shinigami92** [comment]
Please try out the new beta [v1.16.0-0](https://www.npmjs.com/package/@prettier/plugin-pug/v/1.16.0-0) and report anything back :slightly_smiling_face:

**Comment by bitencode** [comment]
Hi @Shinigami92, now I'm on vacation this week, but have a laptop with me 😉.  I haven't found any issues related to the `pugExplicitDiv` feature, other than the release notes say `pubExplicitDiv` instead of `pugExplicitDiv`.

I ran 1.16.0-0 on two of my personal projects and found no issues it all, but both projects have had a single developer and are pretty consistent in style.  One has about 50 components and the other about 75 and `pugExplicitDiv` will correctly add or remove the `div` based on the setting.

I also ran 1.16.0-0 on our SDVI front-end code base and found no issues with `pugExplicitDiv` (worked as expected, found some `div`s that were missed and automatically added them.  I discarded changes and went the other way and it correctly removed all the `div` that had `id` or `class`, so I think we're good 👍 

I did, however find a few issues with some of the output that caused our app to break.  I will create two issues so they can be tracked separately. Out Front-end apps have approximately 500 vue components and have had 15 or so devs contribute over the last few years and for the most part `@prettier/plugin-pug` did a fantastic job of finding and fixing some things that I didn't even notice.  I installed version 1.15.3 and it causes the same issues, so it's not something with version 1.16.
